homeassistant/components/netatmo/data_handler.py

```diff
@@ -31,8 +31,10 @@
     NETATMO_CREATE_CAMERA,
     NETATMO_CREATE_CAMERA_LIGHT,
     NETATMO_CREATE_CLIMATE,
+    NETATMO_CREATE_LIGHT,
```

Comment:
You can probably consolidate all these constants in an Enum

Reply:
Good idea for a future PR.